content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,225 @@
+
+    The output from the `get pod` command should show the following output (`2/2` means that
+    the Sidecar Injector webhook injected a sidecar into the example pod):
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
+
+1.  Check that the validation webhook is working:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-validation
+    $ kubectl apply -n test-validation -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: invalid-gateway
+    spec:
+      selector:
+        # DO NOT CHANGE THESE LABELS
+        # The ingressgateway is defined in install/kubernetes/helm/istio/values.yaml
+        # with these labels
+        istio: ingressgateway
+    EOF
+    {{< /text >}}
+
+    The output from the gateway creation command should show the following output (the error
+    in the output indicates that the validation webhook checked the gateway YAML file):
+
+    {{< text plain >}}
+    Error from server: error when creating "invalid-gateway.yaml": admission webhook "pilot.validation.istio.io" denied the request: configuration is invalid: gateway must have at least one server
+    {{< /text >}}
+
+## `istioctl` shows webhook configurations
+
+1.  Show the configurations of Galley and Sidecar Injector with their default webhook configuration names:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status
+    {{< /text >}}
+
+1.  Show the configuration of Sidecar Injector with the webhook configuration name being `istio-sidecar-injector`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  Show the configuration of Galley with the webhook configuration name being `istio-galley`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## `istioctl` disables webhook configurations
+
+1.  Disable the configurations of Galley and Sidecar Injector with their default webhook configuration names:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable
+    {{< /text >}}
+
+1.  Disable the configuration of Sidecar Injector with the webhook configuration name being `istio-sidecar-injector`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  Disable the configuration of Galley with the webhook configuration name being `istio-galley`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## Cleanup
+
+After completing this tutorial, you may delete the testing cluster created
+at the beginning of this tutorial.
```

Comment:
This is not meaningful information. Can you refer to other tasks to see if you can improve?

Reply:
To safely test an experimental feature, this guide creates a test cluster and the cleanup simply deletes the test cluster. If you want to add individual commands to the guide to revert the commands executed in the guide, please let me know and I can add them.

Reply:
Yes, is there a clean reverting approach?

Reply:
Added the command to delete the resources created.